tests/core/snapd-refresh-vs-services-reboots/task.yaml

```diff
@@ -40,6 +40,12 @@ execute: |
   fi
 
   if [ "$SPREAD_REBOOT" = "0" ]; then
+    # Wait for the usr-lib-snapd.mount unit to be ready, or the installation of
+    # the test snap will fail:
+    # systemd[1]: Dependency failed for Mount unit for test-snapd-svc-flip-flop, revision x1.
+    # systemd[1]: Dependency failed for Service for snap application test-snapd-svc-flip-flop.svc1.
+    retry -n 5 sh -c 'systemctl status usr-lib-snapd.mount | MATCH "active \(mounted\)"'
```

Comment:
yeah I agree with Samuele, this seems suspicious, and especially considering the fact that here, we are executing a known buggy version of snapd this could be peppering over part of the bug behavior.

Then again, the reason this mount unit is not available could actually literally be due to the bug where we had the wrong settings in the unit service files for the snaps, and thus it is sensible to wait for this to ensure that the buggy snapd is at least operating in an expected buggy way rather than a buggy buggy way